libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.AdaptiveDialog.schema

```diff
@@ -44,6 +44,20 @@
             "items": {
                 "$kind": "Microsoft.ITriggerCondition"
             }
+        },
+        "schema": {
+            "anyOf": [
+                {
+                    "$comment": "Eventually we should allow at least some subset of JSON schema here.",
+                    "title": "The schema to be filled in.",
+                    "type": "object",
+                    "additionalProperties": true
+                },
+                {
+                    "type": "string",
+                    "title": "String pointing to schema file."
+                }
```

Comment:
You should be able to do a $ref to the jsonschema definition here.

Reply:
Not sure I understand your comment.  The string is pointing to something like sandwich.schema.dialog so it is location independent.  It could also be an inline schema definition.  I have not yet defined the meta-schema for dialog schema.  (As opposed to component schema.) 

---
In reply to: [360175261](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r360175261) [](ancestors = 360175261)